source/cellular/known-working-3g-4g-modems.rst

```diff
@@ -79,6 +79,10 @@ pfSense® software.
 -  Huawei VIK K3715 HSU by Vodafone
 -  Huawei K3765 by Vodafone
 -  Huawei ME909u-521 4G/LTE Mini-PCIe
+
+   -  Link interface: /dev/cuaU0.0
+   -  Init string: &F0E1Q0 +CMEE=2
```

Comment:
Put the values in backticks so they are displayed as literal values. (` ``blah`` `)

Reply:
Done